flocker/ca/_ca.py

```diff
@@ -229,6 +231,65 @@ def write_credential_files(self, key_filename, certificate_filename):
             os.umask(original_umask)
 
 
+class NodeCredential(FlockerCredential):
+    """
+    A certificate for a node agent, signed by a supplied certificate
+    authority.
+
+    :ivar bytes uuid: A unique identifier for the node this certificate
+        identifies, in the form of a version 4 UUID.
+    """
```

Comment:
I wouldn't have thought so, though worth mentioning the follow up API user certificate branch removes this inheritance and uses composition instead, with the docstring specifying that the `credential` field is a `FlockerCredential`.